src/pages/de/core-concepts/endpoints.mdx

```diff
@@ -0,0 +1,282 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Endpunkte
+description: Erfahre wie du Endpunkte erstellen kannst, die jegliche Form von Daten bereitstellen
+---
+
+Astro lässt Dich unterschiedliche projektspezifische Endpunkte erstellen, die jegliche Form von Daten bereitstellen können. Du kannst sie verwenden um Bilder zu generieren, ein RSS-Dokument bereitzustellen oder die als API-Route verwenden um eine komplette API für Deine Seite bereitzustellen.
+
+In statisch generierten Seiten werden die projektspezifischen Endpunkte zum Erstellungszeitpunkt abgerufen. Wenn du den [SSR](/de/guides/server-side-rendering/)-Modus verwendest, verhalten sich die projektspezifischen Endpunkte wie "echte" API-Endpunkte, die bei jedem Request ausgeführt werden. Statische und SSR-Endpunkte werden auf ähnliche Weise definiert, aber SSR-Endpunkte bieten mehr Optionen.
+
+## Endpunkte für statische Dateien
+
+Um einen Endpunkt zu erstellen, füge eine `.js` oder `.ts`-Datei dem `/pages`-Verzeichnis hinzu. Die Erweiterung `.js` oder `.ts` wird während des Erstellungsprozesses entfernt, so dass der Dateiname die Dateiendung der Daten, die man erzeugen möchte, enthalten sollte. So erzeugt `src/pages/data.json.ts` zum Beispiel den Endpunkt `/data.json`.
+
+Endpunkte exportieren eine `get`-Funktion (optional auch mit `async` Auszeichnung), welche ein [Context-Objekt](/en/reference/api-reference/#endpoint-context) als Parameter übergeben bekommt. Die Inhalte sind ähnlich zu globalen `Astro`-Variable. Die Funktion gibt ein Dokument mit `body`-Objekt zurück, welches Astro zum Erstellungszeitpunkt aufruft. Die Inhalte des Dokuments werden dann zur Erstellung der Datei genutzt.
```

Comment:
In dem Satz "Die Inhalte sind ähnlich zu globalen `Astro`-Variable" würde ich noch ein "der" einfügen, damit es sich besser liest: "Die Inhalte sind ähnlich zu der globalen `Astro`-Variable."